debezium-core/src/main/java/io/debezium/jdbc/JdbcValueConverters.java

```diff
@@ -474,7 +486,7 @@ protected Object convertTimestampToEpochNanos(Column column, Field fieldDefn, Ob
      */
     protected Object convertTimestampToEpochMillisAsDate(Column column, Field fieldDefn, Object data) {
         // epoch is the fallback value
-        return convertValue(column, fieldDefn, data, 0L, (r) -> {
+        return convertValue(column, fieldDefn, data, new java.util.Date(), (r) -> {
```

Comment:
Zero is missing

Reply:
I don't think it's needed; see my comment on your original PR: this check has been done before, and this method only is meant to ever return `Date`. It's never called for the non-Connect modes.

Reply:
@gunnarmorling No, zero in parmaeter, instead of epoch you are sending current timestamp

Reply:
*Gasp*, yes. Fixed.

One more indication that a test *is* missing. Would you be up for pushing one more commit that adds the IT I suggested in the other comment?

Reply:
@gunnarmorling I am just trying to push a one